Software/build-vars.prf.default

```diff
@@ -23,6 +23,11 @@ win32: {
     TARGET_ARCH = x86_64
 }
 
+unix:!macx {
+    DEFINES += SOUNDVIZ_SUPPORT
+    DEFINES += PULSEAUDIO_SUPPORT
+}
```

Comment:
This (and `src.pro`) should probably copy `BASS_SOUND_SUPPORT` parts.
Mac has a short version of this only because it's all first party frameworks that are already in the Mac build core.

Reply:
You mean check if pulseaudio dev package is installed?

Reply:
No (well, I guess that's an option), I mean in top level `build-vars.prf` you'd only define `PULSEAUDIO_SUPPORT` and lib/header `DIR`s, for ex if you link against your own libpulse build from a non-default location. It could be a commented out example that serves as a template.

Then in `src.pro` you'd check for `PULSEAUDIO_SUPPORT` and add your `LIB`s with eventual `DIR`s from above and define `SOUNDVIZ_SUPPORT` kind of like [here](https://github.com/psieg/Lightpack/blob/a8b05af871aed0153dcfb5b2f292cacf189230fc/Software/src/src.pro#L140-L163)